src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py

```diff
@@ -59,10 +59,11 @@ def create_figure(self) -> Figure:
         # # get stacked bar data
         data: dict[str, list[int | float]]
         y_labels: list[str] = []
-        y_axis_label = 'latency [ms]'
+        caption = 'latency'
+        y_axis_label = caption + ' [ms]'
         target_objects = self._metrics.target_objects
         data, y_labels = self._metrics.to_stacked_bar_data()
-        title: str = f"Stacked bar of '{getattr(target_objects, 'path_name')}'"
+        title: str = f'Stacked bar of  Response Time --- {self._case} case ---'
```

Comment:
Together with latency, etc., it should be lower-cased.
```suggestion
        title: str = f'Stacked bar of  response time --- {self._case} case ---'
```